Library/Homebrew/download_strategy.rb

```diff
@@ -455,27 +455,16 @@ def resolve_url_basename_time_file_size(url, timeout: nil)
       url = url.sub(%r{^(https?://#{GitHubPackages::URL_DOMAIN}/)?}o, "#{domain.chomp("/")}/")
     end
 
-    out, _, status= curl_output("--location", "--silent", "--head", "--request", "GET", url.to_s, timeout: timeout)
-
-    lines = status.success? ? out.lines.map(&:chomp) : []
-
-    locations = lines.map { |line| line[/^Location:\s*(.*)$/i, 1] }
-                     .compact
-
-    redirect_url = locations.reduce(url) do |current_url, location|
-      if location.start_with?("//")
-        uri = URI(current_url)
-        "#{uri.scheme}:#{location}"
-      elsif location.start_with?("/")
-        uri = URI(current_url)
-        "#{uri.scheme}://#{uri.host}#{location}"
-      elsif location.start_with?("./")
-        uri = URI(current_url)
-        "#{uri.scheme}://#{uri.host}#{Pathname(uri.path).dirname/location}"
-      else
-        location
-      end
-    end
+    output, _, _status = curl_output(
+      "--location", "--silent", "--head", "--request", "GET", url.to_s,
+      timeout: timeout
```

Comment:
```suggestion
      timeout: timeout,
```

Reply:
Since each parameter isn't on its own line, adding this trailing comma would lead to the following `brew style` error: `Avoid comma after the last parameter of a method call, unless each item is on its own line.`

Reply:
Cool, sorry!